core/src/messages/state.rs

```diff
@@ -8,26 +8,36 @@ use serde::{Deserialize, Serialize};
 use std::net::IpAddr;
 
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
-pub struct WorldStateMessage {
-    pub cluster: ClusterName,
-
-    #[serde(flatten)]
-    pub message: ClusterStateMessage,
+#[serde(untagged)]
+pub enum WorldStateMessage {
+    ClusterMessage {
+        cluster: ClusterName,
+        #[serde(flatten)]
+        message: ClusterStateMessage,
+    },
+    Heartbeat {
+        heartbeat: (),
```

Comment:
maybe have this be a struct along the lines of clustermessage, so
{
message: HeartbeatMessage
}
with 
enum HeartbeatMessage {
None,
}

so that we can change it later if needed